packages/ra-data-simple-rest/src/index.ts

```diff
@@ -10,7 +10,78 @@ import {
     UPDATE_MANY,
     DELETE,
     DELETE_MANY,
-} from 'react-admin';
+} from 'ra-core';
+
+interface Pagenation {
```

Comment:
there already is a Pagination type in ra-core, use that one. Same for sort

Reply:
I'm sorry, I'll check it.